glide.yaml

```diff
@@ -34,7 +36,7 @@ import:
 - package: k8s.io/apiextensions-apiserver
   version: 3eb4c5fefbe7ad866747f78c7e4994cb555170c6
 - package: github.com/prometheus/client_golang
-  version: v0.8.0
+  version: e7e903064f5e9eb5da98208bae10b475d4db0f8c
```

Comment:
Why did you need to update the prometheus client?  should it be using a semantic version still instead of a hash?

Reply:
Prometheus v0.8.0 depends on a different version of client-go than the client-go that rook depends on. By setting it to this hash, prometheus will have the same client-go version dependency than rook thus eliminating any conflict. v0.8.0 is the latest released version for prometheus. Im hoping that by updating this, i didnt cause issues with prometheus.